components/Stats.vue

```diff
@@ -195,7 +195,7 @@
         class="DataCard"
       >
         <time-line-chart-county-comparison
-          :title="`Cases per 1,000 People`"
+          :title="`New Cases per 100,000 Residents 14 Day Average`"
```

Comment:
Maybe some sort of separation for 14 Days Average like in parentheses or dash? `(14 Days Average)` That way it doesn't sound like a run-on sentence.

Reply:
Good catch! Fixed it. I changed the text to make it more clear. I also see the run-on sentence issue on many cards so changed the max-width within the data card to show in one line if it can.